packages/@lwc/engine/src/framework/def.ts

```diff
@@ -154,6 +158,7 @@ function createComponentDef(
         name,
         wire,
         track,
+        fields,
```

Comment:
this is only needed if you need to know what are the fields later on... which I don't think you need to, you just install the descriptor and forget about them. remove it.